pkg/storage/inmemory/article.go

```diff
@@ -0,0 +1,97 @@
+package inmemory
+
+import (
+	"errors"
+	"github.com/reiver/go-porterstemmer"
+	"news-aggregator/pkg/model"
+	"strings"
+	"time"
+)
+
+// ArticleInMemory is a struct that contains the in-memory database for articles.
+type ArticleInMemory struct {
+	Articles []model.Article
```

Comment:
instead of this, you can add a map where you will store data

Reply:
I've decided to use slice, because I prefer to keep articles order